daemon/logger/journald/read.go

```diff
@@ -245,7 +251,7 @@ drain:
 	return cursor
 }
 
-func (s *journald) followJournal(logWatcher *logger.LogWatcher, config logger.ReadConfig, j *C.sd_journal, pfd [2]C.int, cursor *C.char) *C.char {
+func (s *journald) followJournal(logWatcher *logger.LogWatcher, config logger.ReadConfig, j *C.sd_journal, pfd [2]C.int, cursor *C.char, untilUnixMicro uint64) *C.char {
```

Comment:
I think this a bit different because `config.Until` is parsed in a specific way. I could extract all that to a common function but I think that would result in more boilerplate code than less.